management/management_api/endpoints/endpoint_utils.py

```diff
@@ -166,21 +164,20 @@ def view_endpoint(endpoint_name: str, namespace: str, id_token: str):
 
     endpoint_status = get_endpoint_status(api_instance=api_instance, namespace=namespace,
                                           endpoint_name=endpoint_name)
-    model_path = create_url_to_service(endpoint_name, namespace)
+    endpoint_url = create_url_to_service(endpoint_name, namespace)
     subject_name, resources = get_crd_subject_name_and_resources(
         custom_api_instance=custom_api_instance, namespace=namespace, endpoint_name=endpoint_name)
     replicas = get_replicas(apps_api_instance=apps_api_instance, namespace=namespace,
                             endpoint_name=endpoint_name)
 
-    view_dict = {'Endpoint status': endpoint_status,
-                 'Model path': model_path,
-                 'Subject name': subject_name,
-                 'Resources': resources,
-                 'Replicas': replicas,
+    view_dict = {'endpoint status': endpoint_status,
+                 'endpoint url': endpoint_url,
+                 'subject name': subject_name,
+                 'resources': resources,
+                 'replicas': replicas,
                  }
-    message = f"Endpoint {endpoint_name} in {namespace} tenant: {view_dict}\n"
-    logger.info(message)
-    return message
+    logger.info("Endpoint {} in {} tenant: {}".format(endpoint_name, namespace, view_dict))

```

Comment:
Why did you switch to `.format()`. I thought we want to have f-strings in management api.